src/main/resources/assets/admin/common/js/util/htmlarea/editor/HTMLAreaBuilderCKE.ts

```diff
@@ -245,14 +245,17 @@ module api.util.htmlarea.editor {
         private handleFileUpload(ckeditor: HTMLAreaEditor) {
             ckeditor.on('fileUploadRequest', (evt: eventInfo) => {
                 const fileLoader = evt.data.fileLoader;
-                const formData = new FormData();
-                const xhr = fileLoader.xhr;
 
-                xhr.open('POST', fileLoader.uploadUrl, true);
-                formData.append('file', fileLoader.file, fileLoader.fileName);
-                formData.set('parent', this.content.getPath().toString());
-                formData.set('name', fileLoader.fileName);
-                fileLoader.xhr.send(formData);
+                this.isFileExisting(fileLoader.fileName).then((exists: boolean) => {
+                    if (exists) {
+                        NotifyManager.get().showWarning(`File ${fileLoader.fileName} already exists!`);
```

Comment:
This warning should be localised/translated

Reply:
Ok!

Reply:
https://github.com/enonic/app-contentstudio/pull/193